posts/faq/how-to-backup-wallet.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How do I mine backup my wallet?
+---
+
+You can use `lbrycrd-cli backupwallet <pathtobackup>` to back up your wallet, or alternatively you can copy the wallet.dat file manually to anywhere you choose [important directories](https://lbry.io/faq/lbry-directories).
```

Comment:
i like the links to "important directories" everywhere, but you should either link to it as part of the sentence, or put it on a separate line. For example:

> You can use `lbrycrd-cli backupwallet <pathtobackup>` to back up your wallet, or alternatively you can copy the wallet.dat file manually to anywhere you choose.

> See also: [important directories](https://lbry.io/faq/lbry-directories)